src/app/components/fees-row/fees-row.tsx

```diff
@@ -63,15 +63,14 @@ export function FeesRow({
   }, [feeHelper, defaultFeeValue, feeTypeHelper]);
 
   useEffect(() => {
-    if (!defaultFeeValue && hasFeeEstimates && !feeField.value && !isCustom) {
+    if (isSponsored) {
+      void feeHelper.setValue(0);
+    } else if (!defaultFeeValue && hasFeeEstimates && !feeField.value && !isCustom) {
```

Comment:
Thoughts on [early return pattern](https://gomakethings.com/the-early-return-pattern-in-javascript/) and avoiding else if?

Reply:
Done 👍